src/App.jsx

```diff
@@ -14,57 +16,59 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [selectedGood, setSelectedGood] = useState('Jam');
```

Comment:
The initial state of `selectedGood` is set to 'Jam'. If the requirement is to start with no selection, consider initializing it with an empty string ('') instead.